packages/loader/README.md

```diff
@@ -1,11 +1,9 @@
 # [`@fremtind/jkl-loader`](https://fremtind.github.io/jokul/components/loader/)
 
-## Om spinneren
+Lasteindikatoren viser at noe er underveis / at systemet jobber. Vi bruker den dersom en handling tar lengre tid enn to - tre sekunder. 
```

Comment:
"Lasteindikatoren viser at systemet jobber og brukes hvis en handling tar lengre tid enn to – tre sekunder."

Reply:
oppdatert